ci/install-package.sh

```diff
@@ -29,6 +29,17 @@ do
 
         -r=* | --pullrequest=* )
             PULL_REQUEST="${i#*=}" ;;
+
+        -i=* | --image=* )
+            IMAGE_NAME="${i#*=}" ;;
+
+        -s=* | --ssh=* )
+            USE_SSH="${i#*=}"
+            USE_SSH=${USE_SSH:-"false"} ;;
+
+        -sk=* | --ssh-key=* )
```

Comment:
`-sk` is against the arguments convention.

Reply:
Is it supposed to be just 1 letter argument?

Reply:
one dash should be followed by one letter, In case of multiple letters, multiple dashed should be used.

Reply:
Fixed